glommio/src/sync/rwlock.rs

```diff
@@ -126,10 +125,10 @@ impl From<TryLockError> for io::Error {
     }
 }
 
-struct Waiter<T> {
+struct Waiter {
```

Comment:
Looks better to my eyes.

@matklad I'll give you a chance to comment before settling, though.

If any of you know anybody who works on github, please beg them to implement comments in the commit message. This comment is about the entire diff, not about the line in which I had to attach it!!